pipelines/managed/fbc-release/fbc-release.yaml

```diff
@@ -155,6 +147,42 @@ spec:
           workspace: release-workspace
       runAfter:
         - verify-access-to-resources
+    - name: collect-task-params
```

Comment:
I want to clarify something. I thought that the idea is that we would use the new generic task in place of the specific ones - I automatically assumed we would have N instances of the task per pipeline. In that case the name would be something like `collect-conforma-params`. But because you can specify numerous keys to extract, we can probably have just one instance per pipeline, right? And in that case the name `collect-task-params` makes sense.

Reply:
My plan (which will be shown in the future PRs where I remove each collect-*-params task) is to have this collect-task-params task called just once with a bunch of values extracted